HIRS_Utils/src/main/java/hirs/data/persist/Alert.java

```diff
@@ -45,6 +45,9 @@
     @Column(name = "device_name")
     private String deviceName;
 
+    @Column(name = "display_title", nullable = true)
```

Comment:
Do we have a reason we prefer null as a default vs. an empty string? Empty strings are less troublesome for JSPs and other code.

Reply:
only one comment

Reply:
@apldev1 any thoughts on this?

Reply:
Mostly the preference is to use null to signal that there is no content in the field, as only one alert type currently sets a value for the field.  The Portal behavior changes based on whether the field has content or not, so I figured it the code would clearly illustrate this by checking for null (alternatively, we'd still have to check for an empty string.)  I'm open to dissenting opinions, though.

Reply:
Can we get a decision on this and get this merged? I'm fine leaving it as is

Reply:
I'm fine with leaving it as `null`, people can check for it.